### PR TITLE
chore: Changed samples in Bigtable to new format for the 'list samples' - issue #1389

### PR DESCRIPTION
**Pull request by saranshdhingra**

### CODE REVIEW

File: bigtable/src/list_instance.php [hunk 1]
```diff
@@ -25,30 +25,30 @@
 // Include Google Cloud dependencies using Composer
 require_once __DIR__ . '/../vendor/autoload.php';
```
Comment (bshaffer) [inline bigtable/src/list_instance.php:26]: As we discussed, we should remove all these!

File: bigtable/src/list_instance.php [hunk 2]
```diff
@@ -25,30 +25,30 @@
...
-/** Uncomment and populate these variables in your code */
-// $projectId = 'The Google project ID';
-// $instanceId = 'The Bigtable instance ID';
-
-$instanceAdminClient = new BigtableInstanceAdminClient();
+/**
+ * List Bigtable instances in a project
+ * @param string $projectId The Google Cloud project ID
+ */
+function list_instance($projectId)
```
Comment (bshaffer) [inline bigtable/src/list_instance.php:35]: Let's add typehints for all these
```suggestion
function list_instance(string $projectId): void
```

**Review by bshaffer — COMMENTED** [review]
Really great work! Just a few suggestions.

File: bigtable/test/bigtableTest.php [hunk 3]
```diff
@@ -109,8 +109,7 @@ public function testCreateDevInstance()
     public function testListInstances()
     {
         $content = self::runSnippet('list_instance', [
-            self::$projectId,
-            self::$instanceId
+            self::$projectId
```
Comment (saranshdhingra) [inline bigtable/test/bigtableTest.php:112]: Just wanted to highlight the removal of this as the instanceId wasn't being used by the sample anyway.

**Review by bshaffer — APPROVED** [review]

**Pull request merged**